.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,31 @@
+name: Build wheels
+
+on:
+  workflow_dispatch
+
+jobs:
+  buildwheels:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.6]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: install automake
+      run: brew install automake
+      if: matrix.os == 'macOS-latest'
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - name: install cibuildwheel
+      run: python -m pip install cibuildwheel==2.2.0a1
```

Comment:
Why are we depending on the alpha release? Is there a specific feature that we need that isn't available in one of the normal releases?

Reply:
I copy-pasted this from the cibuildwheels documentation. I'll see if we can get away with the latest released version (2.3.1).